src/test/app/ValidatorList_test.cpp

```diff
@@ -570,6 +570,25 @@ class ValidatorList_test : public beast::unit_test::suite
             BEAST_EXPECT(trustedKeys->listed(val.signingPublic));
         }
 
+        const auto hexPublic =
+            strHex(publisherPublic.begin(), publisherPublic.end());
+
+        const auto available = trustedKeys->getAvailable(hexPublic);
+
+        BEAST_EXPECT(available);
+        if (available)
+        {
+            BEAST_EXPECT(
```

Comment:
You're missing an include for the non-unity builds, so `jss` is unknown. You need `#include <ripple/protocol/jss.h>`